packages/node_modules/@webex/widget-meet/src/components/call-active/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-import {WebexInMeeting, WebexMeetingControls, WebexMeetingControl, WebexDataProvider} from '@webex/components';
+import {WebexInMeeting, WebexMeetingControls, WebexMeetingControl, WebexDataProvider} from '@webex/components/dist/es/webex-components.es';
```

Comment:
I think this is ok for debugging, but we should let webpack decide which bundle it will import.
```suggestion
import {WebexInMeeting, WebexMeetingControls, WebexMeetingControl, WebexDataProvider} from '@webex/components';
```